cmd/fleet/cron.go

```diff
@@ -666,6 +666,12 @@ func startCleanupsAndAggregationSchedule(
 				return enrollHostLimiter.SyncEnrolledHostIDs(ctx)
 			},
 		),
+		schedule.WithJob(
+			"cleanup_host_operating_systems",
+			func(ctx context.Context) error {
+				return ds.CleanupHostOperatingSystems(ctx)
+			},
```

Comment:
```suggestion
			ds.CleanupHostOperatingSystems,
```

Reply:
@chiiph, I see your point and agree that the anonymous function isn't strictly needed here since the underlying function has the same signature, but it is the pattern we've followed generally in the scheduler. I wonder if the extra verbosity helps to signal anything to future readers about how scheduled jobs can be defined. @lucasmrod do you have any thoughts on the pattern?

Reply:
Let's keep as-is (which is similar to the surrounding code)